README.md

```diff
@@ -9,9 +9,29 @@ Also included:
   + APIs for packet data over radio [README_data](README_data.md)
   + An STM32 embedded version of FreeDV 1600/700D/700E for the [SM1000](stm32/README.md)
 
-## Older code
+## Old Code and Deprecated Features
 
-In July 2023 this repo was refactored, older code can be found in https://github.com/drowe67/codec2-dev
+In July 2023 this repo was refactored, older code can be found in https://github.com/drowe67/codec2-dev. 
+
+We are currently conducting a major re-development of Codec 2, new speech coding, modems, and FreeDV modes are under active development.  We have limited resources available for maintenance or development on features that may soon be replaced (with the exception of major bugs).  We'd rather put our efforts into new features! Modes and Features we are not actively maintaining at present, and likely to be superseded in the near future include:
+1. FreeDV 1600, 700C, 700D, 700E, 2020.
+1. Codec 2 modes with the exception of Codec 2 3200 for M17.
```

Comment:
Per page 28 of https://spec.m17project.org/, Codec2 1600 is also used.

```suggestion
1. Codec 2 modes with the exception of Codec 2 1600 and 3200 for M17.
```

Reply:
Woj has specifically asked us to improve 1600, so would deprecating/not maintaining the current 1600 be one of our aims?

Reply:
Fair enough, we can keep as is.